Doc/library/os.rst

```diff
@@ -569,6 +569,38 @@ process and user.
       See the documentation for :func:`getgroups` for cases where it may not
       return the same group list set by calling setgroups().
 
+.. function:: setns(fd, nstype=0)
+
+   Reassociate the current with a namespace.
```

Comment:
```suggestion
   Reassociate the current thread with a `Linux namespace
   <https://en.wikipedia.org/wiki/Linux_namespaces>`_.
```

* Add back missing word "thread" (I assume that's correct, from reading the `setns` manpage)?
* Add "Linux" to namespace to clarify the term and avoid confusion with its much more common use in Python
* Explicit link the term namespace to an external definition/explanation to further reduce ambiguity and provide users an easily accessible resource for a high-level summary of the main concept. Should we link something else other than what I suggested here, though?